go-controller/pkg/ovn/pod_selector_address_set.go

```diff
@@ -467,7 +469,10 @@ func (bnc *BaseNetworkController) podSelectorPodNeedsDelete(pod *kapi.Pod, podHa
 	}
 	ips, err := util.GetPodIPsOfNetwork(pod, bnc.NetInfo)
 	if err != nil {
-		return "", fmt.Errorf("can't get pod IPs %s/%s: %w", pod.Namespace, pod.Name, err)
+		// if pod has no IP, nothing to do
+		klog.Warningf("Failed to get IPs of pod %s/%s during address_set pod selector removal: %v",
+			pod.Namespace, pod.Name, err)
+		return "", nil
```

Comment:
It looks like `GetPodIPsOfNetwork` wouldn't need to return an error for anything.

Reply:
not sure what you mean

Reply:
The only reason for GetPodIPsOfNetwork to return an error is because pod status IP is empty. It could just return `nil` in that case.